tests/rados/test_bluestoretool_workflows.py

```diff
@@ -0,0 +1,293 @@
+            cmd=f"cephadm shell -- ceph-volume lvm list {osd_id} --format json",
+        )
+        lvm_json = json.loads(lvm_list)
+        dev = lvm_json[f"{osd_id}"][0]["tags"]["ceph.block_device"]
+        log.info(
+            f"\n --------------------"
+            f"\n Generate the content for an OSD data directory "
+            f"that can start up a BlueStore OSD for OSD {osd_id}"
+            f"\n --------------------"
+        )
+        out = bluestore_obj.generate_prime_osd_dir(osd_id=osd_id, device=dev)
+        log.info(out)
+
+        # Execute ceph-bluestore-tool free-dump --path <osd_path> [--allocator block/bluefs-wal/bluefs-db/bluefs-slow]
+        osd_id = random.randrange(len(osd_list))
+        log.info(
+            f"\n --------------------"
+            f"\n Dump all free regions in allocator for OSD {osd_id}"
+            f"\n --------------------"
+        )
+        out = (
+            bluestore_obj.get_free_dump(osd_id=osd_id)
+            if rhbuild.startswith("6")
+            else bluestore_obj.get_free_dump(osd_id=osd_id, allocator_type="block")
+        )
+        log.debug(out)
+        assert "alloc_name" in out and "extents" in out
+
+        # Execute ceph-bluestore-tool free-score --path <osd_path> [--allocator block/bluefs-wal/bluefs-db/bluefs-slow]
+        osd_id = random.randrange(len(osd_list))
+        log.info(
+            f"\n --------------------"
+            f"\n Get fragmentation score for OSD {osd_id}"
+            f"\n --------------------"
+        )
+        out = (
+            bluestore_obj.get_free_score(osd_id=osd_id)
+            if rhbuild.startswith("6")
+            else bluestore_obj.get_free_score(osd_id=osd_id, allocator_type="block")
+        )
+        log.info(out)
+        assert "fragmentation_rating" in out
+
+        # Execute ceph-bluestore-tool show-sharding --path <osd_path>
+        osd_id = random.randrange(len(osd_list))
+        log.info(
+            f"\n --------------------"
+            f"\n Show sharding that is currently applied to "
+            f"BlueStore’s RocksDB for OSD {osd_id}"
+            f"\n --------------------"
+        )
+        out = bluestore_obj.show_sharding(osd_id=osd_id)
+        log.info(out)
+        assert "block_cache" in out
+
+        # Execute ceph-bluestore-tool bluefs-stats --path <osd_path>
+        osd_id = random.randrange(len(osd_list))
+        log.info(
+            f"\n --------------------"
+            f"\n Display BlueFS statistics for OSD {osd_id}"
+            f"\n --------------------"
+        )
+        out = bluestore_obj.show_bluefs_stats(osd_id=osd_id)
+        log.info(out)
+        assert (
+            "device size" in out
+            and "wal_total" in out
+            and "db_total" in out
+            and "slow_total" in out
+        )
+
+        # restart OSD services
+        osd_services = rados_obj.list_orch_services(service_type="osd")
+        for osd_service in osd_services:
```

Comment:
This was not required. I was just speaking about the start and stop that was performed for running CBT tool.

But no harm in adding this check as well.